README.md

```diff
@@ -92,8 +92,8 @@ jedis.sadd("planets", "Mars");
 
 ## Using Redis modules
 
-Jedis provides support for some of the [Redis modules](https://redis.io/docs/modules/), most notably
-[RedisJSON](https://oss.redis.com/redisjson/) and [RediSearch](https://oss.redis.com/redisearch/).
+Jedis includes support for [Redis modules](https://redis.io/docs/modules/) such as
+[RedisJSON](https://oss.redis.com/redisjson/) and [RediSearch](https://oss.redis.com/redisearch/). For ease of high-level JSON and Search interactions, you might want to see the [redis-om-spring](https://github.com/redis/redis-om-spring) client.
```

Comment:
> Are you looking for a high-level library to handle object mapping? See [redis-om-spring](https://github.com/redis/redis-om-spring)!

We have already included above line. The changed line seems redundant.

We can consider removing the existing line.

Reply:
Agreed. Removed the dupe.